main_routers/live2d_router.py

```
@@ -407,7 +529,34 @@ async def update_emotion_mapping(model_name: str, request: Request):
         file_refs['Expressions'] = preserved_expressions + new_expressions
 
         # 同时保留一份 EmotionMapping（供管理器读取与向后兼容）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)